src/warnet/cli/network.py

```diff
@@ -24,9 +25,12 @@ def start(graph_file: Path, force: bool, network: str):
     Start a warnet with topology loaded from a <graph_file> into <--network> (default: "warnet")
     """
     try:
+        encoded_graph_file = ""
+        with open(str(graph_file), "rb") as graph_file_buffer:
```

Comment:
```suggestion
        with open(graph_file, "rb") as graph_file_buffer:
```

Don't think we should need the `str()` here? `open` can handle `Path`

Reply:
Removed str()